Components/Cards/cards.css

```diff
@@ -0,0 +1,45 @@
+/* cards */
+.cart-versions {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.cart {
+    margin: 1rem;
+    padding: 1rem;
+    max-width: 100%;
+    font-size: 10px;
+}
+
+.img-contents {
```

Comment:
You can rename `.img-contents` to something useful like `card`
These can also have some more padding.